public/locales/en-US/common.json

```diff
@@ -35,25 +46,23 @@
 				"title": "Diversity",
 				"description": "Our community is made up of all types of people, who provide a different point of view on a problem, always leading us to different discussions and new learnings."
 			}
+		},
+		"description": {
+			"title": "With space for everyone hungry for knowledge",
+			"subtitle": {
+				"subtitleBegin": "What is ",
+				"subtitleTechmmunity": "<COMPANY_NAME>",
+				"subtitleEnd": " made of?"
+			}
 		}
```

Comment:
`description` deve ficar a cima dos `cards` e no `subtitle` deve ter apenas `What is <COMPANY_NAME> made of?`. Exemplo: `"subtitle": "What is <COMPANY_NAME> made of?"`

Reply:
acima*